packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart

```diff
@@ -94,7 +91,7 @@ class TimelineScreenBodyState extends State<TimelineScreenBody> {
               firstChild: TimelineFlameChart(),
               // TODO(kenz): use StreamBuilder to get selected event from
```

Comment:
Is this TODO still needed now that we're just retrieving the data from the controller?

Reply:
removed